src/reference/config/network-params.md

```diff
@@ -0,0 +1,86 @@
+# Network Parameters
+
+## `network.address` {#param-address}
+
+- **Type:** String, [Socket-Address](glossary#type-socket-address)
+- **Required**
+
+Address for p2p communication for consensus (sumeragi) and block synchronization (block_sync) purposes.
+
+```toml
+[network]
+address = "localhost:1337"
+```
+
```

Comment:
I guess new param should go somewhere here: 

```suggestion
## `network.idle_timeout` {#param-idle-timeout}

- **Type:** String or Number, [Duration](glossary#type-duration)
- **Default:** 60 seconds

Duration of time after which connection with peer is terminated if peer is idle.

```toml
[network]
idle_timeout = "60s"
```
```